modules/tinymce/src/plugins/advlist/main/ts/ui/Buttons.ts

```diff
@@ -36,6 +36,14 @@ const makeSetupHandler = (editor: Editor, nodeName: ListType) => (api: Toolbar.T
 };
 
 const addSplitButton = (editor: Editor, id: string, tooltip: string, cmd: string, nodeName: ListType, styles: string[]): void => {
+  const listStyleTypeAliases: Record<string, string> = {
+    'lower-latin': 'lower-alpha',
+    'upper-latin': 'upper-alpha',
+    'lower-alpha': 'lower-latin',
+    'upper-alpha': 'upper-latin'
```

Comment:
```suggestion
  const listStyleTypeAliases: Record<string, string[]> = {
    'lower-latin': [ 'lower-alpha' ],
    'upper-latin': [ 'upper-alpha' ],
    'lower-alpha': [ 'lower-latin' ],
    'upper-alpha': [ 'upper-latin' ]
```
Would there be any benefit to making these string arrays? In case there are any more aliases added in the future?

Reply:
Maybe deal with that if it ever happens also web standard these days is more strict so it will likely not happen as often as before when browsers just implemented what ever they felt like.